Source/EventFlow.ReadStores.ElasticSearch/EsReadModelStore.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using EventFlow.Aggregates;
+using EventFlow.Logs;
+using Nest;
+
+namespace EventFlow.ReadStores.Elasticsearch
+{
+    public class EsReadModelStore<TAggregate, TReadModel> : ReadModelStore<TAggregate, TReadModel>, 
+                                                            IEsReadModelStore<TAggregate, TReadModel> 
+        where TReadModel : class, IEsReadModel, new() 
+        where TAggregate : IAggregateRoot
+    {
+        private readonly IElasticClient _elasticClient;
+
+        public EsReadModelStore(IElasticClient elasticClient, ILog log) : base(log)
+        {
+            _elasticClient = elasticClient;
+        }
+
+       public override async Task UpdateReadModelAsync(
+            string aggregateId,
+            IReadOnlyCollection<IDomainEvent> domainEvents,
+            CancellationToken cancellationToken)
+       {
+           var readModelResponse = await _elasticClient.GetAsync<TReadModel>(aggregateId)
+               .ConfigureAwait(false);
+
+           var readModel = readModelResponse.Source ??
+                           new TReadModel
+                           {
+                               AggregateId = aggregateId,
+                               CreateTime = DateTimeOffset.Now,
+                               UpdatedTime = DateTimeOffset.Now
+                           };
+
+           Log.Debug("ReadModel: " + readModel);
```

Comment:
@tabro Use format string built into the `Debug` method. I'll look at getting ReSharper support for it and it might be possible without added the NuGet reference.